File: Capstone/Capstone/Models/usuario.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace Capstone.Models
{
    [Table("usuario")]
    public class Usuario
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Column("usuario_id")]
        public long Usuario_id { get; set; }
        [Required]
        [StringLength(200)]
        [Column("usuario")]
        public string? User { get; set; }
        [Required]
        [StringLength(200)]
        [Column("contraseña")]
        public string? Contraseña { get; set; }
        [Required]
        [StringLength(200)]
        [Column("cargo")]
        public string? Cargo { get; set; }
        [Required]
        [StringLength(200)]
        [Column("perfil")]
        public string? Perfil { get; set; }
        [Required]
        [Column("fecha_registro")]
        public DateTime Fecha_registro { get; set; }
        public Usuario() { }
        public Usuario(long Usuario_id, string Usuario, string Contraseña, string Cargo, string Perfil, DateTime Fecha_registro)
        {
            this.Usuario_id = Usuario_id;
            User = Usuario;
            this.Contraseña = Contraseña;
            this.Cargo = Cargo;
            this.Perfil = Perfil;
            this.Fecha_registro = Fecha_registro;

        }
    }
}

File: Proyecto Vecinos Unidos/models/usuario.py
from flask import Flask, request, jsonify
from datetime import datetime
from flask_sqlalchemy import SQLAlchemy
from werkzeug.security import generate_password_hash



db = SQLAlchemy()

# Modelo para la tabla usuario
class Usuario(db.Model):
    __tablename__ = 'usuario'
    usuario_id = db.Column(db.Integer, primary_key=True)
    usuario = db.Column(db.String(100), nullable=False)
    contraseña = db.Column(db.String(100), nullable=False)
    cargo = db.Column(db.String(210),nullable=False)
    perfil = db.Column(db.String(210),nullable=False)
    fecha_registro = db.Column(db.DateTime, default=datetime.utcnow)  # Campo con fecha actual por defecto

    def set_password(self, password):
        self.password = generate_password_hash(password)